services/autoscaling/requirements/_test.in

```diff
@@ -22,9 +23,11 @@ moto[server]
 psutil
 pytest
 pytest-asyncio
+pytest-clarity
```

Comment:
thisone is new!
perhaps we can make a list of pytest-* tools and install them in all requirements/_test.in

Reply:
actually I removed it.